Debugging documentation (#254)
utopia-rise/godot-kotlin-jvm: docs/src/doc/user-guide/debugging.md

## Prerequisites

> If you want to debug your code and you're already using an embedded jre, make sure you've included the following modules:

Reviewer: ```suggestion
In order to debug your code using an embedded JRE, make sure to include the following modules:
```

- `jdk.jdwp.agent`

!!! info
    Example: `jlink --add-modules java.base,java.logging,jdk.jdwp.agent --output jre`

> If you still rely on your installed jdk in `JAVA_HOME` you can just follow on.

Reviewer: nit: probably we don't need to mention this.
